eventtracking/django/models.py

```diff
@@ -0,0 +1,125 @@
+"""
+Models for filtering of events
+"""
+import logging
+import re
+
+from django.core.exceptions import ValidationError
+from django.db import models
+
+logger = logging.getLogger(__name__)
+
+
+def validate_regex_list(value):
+    """
+    Validate every regular expression in the value by trying to compile
+    it.
+
+    Raise Validation error if any reg exp is failed to compile.
+    """
+    invalid_regex_expressions = []
+    expressions = value.split(',')
+    raw_expressions = [expression.strip() for expression in expressions]
```

Comment:
move validate_regex_list  to  `RegExFilter` and use `compiled_expressions` property to remove duplicate code.

Reply:
If we move this validator method inside the model, we won't be able to provide this method in the validators list. Also,`compiled_expressions` property uses `self.regular_expressions` property and this validator is supposed to actually validate that property.

Pardon me if I didn't understand your point correctly.

Reply:
Well I want to avoid the duplication of code by any means.

Reply:
I've added some helper method for the purpose/